src/style.css

```diff
@@ -0,0 +1,107 @@
+/* stylelint-disable CssSyntaxError */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable color-hex-length */
+/* stylelint-disable color-hex-case */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search-form {
```

Comment:
You have a lot of repeatable styles. 
So you need to refactor you code. I will give you an EXAMPLE, how to do it

Following the BEM-methodology - use the `block` selector for the FIRST input (`class="form__input"`), where you apply common input styles

```
.input {
  {{common_input_styles}}
}
```

And then you apply `modifier` to the SECOND input (`class="form__input form__input--small"`) that have ONLY differences with base class input
```
.form__input--small {
  {{different_input_styles}}
}
```
So in this way you follow DRY (Don't repeat yourself) principe
Apply this principe all over the styles